test/hotspot/jtreg/runtime/os/TestTracePageSizes.java

```diff
@@ -157,8 +220,6 @@ public static void main(String args[]) throws Exception {
         // Check if debug printing is enabled.
         if (args.length > 0 && args[0].equals("-debug")) {
             debug = true;
-        } else {
-            debug = false;
```

Comment:
Let's keep the code shape with explicit initialization here? Also drop the init at the field declaration?

Reply:
I did this to quickly turn on debug by changing the default value above. I could not figure out how to add command line parameters to the test on the jtreg command line and did not feel like modifying every single at-run. But I have no strong emotions, if you prefer I can reinstate the old behaviour.

Reply:
Yeah, let's keep it the old way. You can change the `debug = true` right here next time, I think. IIRC, that's what I did before.